graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/MoreSearchAdapterES7.java

```diff
@@ -141,7 +140,13 @@ public void scrollEvents(String queryString, TimeRange timeRange, Set<String> af
                              List<UsedSearchFilter> filters, int batchSize, ScrollEventsCallback resultCallback) throws EventProcessorException {
         final ChunkCommand chunkCommand = buildScrollCommand(queryString, timeRange, affectedIndices, filters, streams, batchSize);
 
-        final ChunkedResult chunkedResult = multiChunkResultRetriever.retrieveChunkedResult(chunkCommand);
+        final ChunkedResult chunkedResult;
+        try {
+            chunkedResult = multiChunkResultRetriever.retrieveChunkedResult(chunkCommand);
+        } catch (IllegalArgumentException e) {
+            LOG.warn("Ignoring invalid scroll request", e);
```

Comment:
Could you please explain why we catch the exception here and don't propagate it further?

Reply:
Added comment in code.
This is intended to allow aggregation or correlation event queries to complete, in the presence of unavailable indices. This was another issue uncovered in unit testing.

Reply:
@patrickmann - I'm sorry if I am saying stupid things, but I do not understand this one.
`MultiChunkResultRetriever` has 4 possible implementations, and you have added options to all 4 of them. 
Why is it still throwing exceptions and requires to handle them manually?

Reply:
It seems that scroll search, when executed on unavailable indices, would not return response with `_scroll_id` field.
Because of that construction will fail on this line:
https://github.com/Graylog2/graylog2-server/blob/813203263b06dda18e2aed68ae92b34277f904b4/graylog-storage-opensearch2/src/main/java/org/graylog/storage/opensearch2/ScrollResultOS2.java#L48

Reply:
@luk-kaminski 
You are right - when scroll searching, the exception is thrown from the constructor you mention; not because of a missing `ignoreUnavailable` option. 
Is there a more elegant way of handling that? I was hesitant to make changes to scroll result processing.